tests/test_sitemap.py

```diff
@@ -69,4 +70,18 @@ def test_sitemap_cache(self):
         ext_key = get_cache_key(page3)
         page3.delete()
         self.assertEqual(cache.get(ext_key), None)
-        
\ No newline at end of file
+
+    @skipIf(not is_versioning_enabled(), 'Right now this feature wont work without versioning')
```

Comment:
Change this to:

@skipIf(not is_versioning_enabled(), 'This test can only run when versioning is installed')